library/options.inc.php

```diff
@@ -207,6 +207,13 @@ function generate_select_list($tag_name, $list_id, $currvalue, $title, $empty_na
 	else {
 		$s .= "</select>";
 	}
+    // add default selected to 'Unassigned' for multiple select
+    if(!$got_selected && !$got_selected_backup && $multiple) {
+
+        $s = str_replace("value=''", "value='' selected ", $s);
```

Comment:
Not an expected practice to scan/replace the output. Better to mark as selected in the code above.